### PR TITLE
add system field

### PR DESCRIPTION
**Pull request by darkonie**
add a new field to node health report - system
"system": {
  "memory": ...
  "load_avarage": ...
  "partitions": ...
  "disk_usage": ...
}

### CODE REVIEW

File: api/health.go [hunk 1]
```diff
@@ -285,8 +288,54 @@ func normalizeProperty(unitName string, p map[string]interface{}, d DCOSHelper)
 	}
 }
 
+func updateSystemMetrics() (sysMetrics sysMetrics, err error) {
```
Comment (malnick) [inline api/health.go]: You could possibly simplify/clarify this block a bit by adding the types from psutil lib directly to your parent struct like:

```
type sysMetrics struct {
	Memory *mem.VirtualMemoryStat  `json:"memory"`
	CPU    []cpu.CPUInfoStat       `json:"cpu"`
	Host   *host.HostInfoStat      `json:"host"`
	Load   *load.LoadAvgStat       `json:"load"`
	Disk   *disk.DiskUsageStat     `json:"disk"`
```

And then declaring getter funcs on it:

```
func HostInfo() *host.HostInfoStat {
	h, _ := host.HostInfo()
	return h
}

func HostMemory() *mem.VirtualMemoryStat {
	memory, _ := mem.VirtualMemory()
	return memory
}

func HostCpu() []cpu.CPUInfoStat {
	c, _ := cpu.CPUInfo()
	return c
}

func HostLoad() *load.LoadAvgStat {
	l, _ := load.LoadAvg()
	return l
}
```

Comment (malnick) [inline api/health.go]: ^^ Not a blocker, just something to consider without taking into account the overall structure of implementation.

File: api/health.go [hunk 2]
```diff
@@ -318,15 +367,17 @@ func GetUnitsProperties(config *Config) (UnitsHealthResponseJSONStruct, error) {
...
-		MesosID:     config.DcosTools.GetMesosNodeID(config.DcosTools.GetNodeRole(), "id"),
-		TdtVersion:  config.Version,
-	}, nil
+		return healthReport, err
+	}
+
+	// update the rest of healthReport fields
+	healthReport.Array = allUnitsProperties
+	healthReport.Hostname = config.DcosTools.GetHostname()
+	healthReport.IPAddress = config.DcosTools.DetectIP()
```
Comment (malnick) [inline api/health.go]: 👍 I like the `.DcosTools` nameing convention. Might want to consider `DCOSTools` following general go linter guidelines.

**Comment by malnick** [comment]
A few comments but nothing major.

File: api/health.go (hunk 1 above)
Comment (darkonie) [inline api/health.go]: done

File: api/health.go (hunk 2 above)
Comment (darkonie) [inline api/health.go]: done

**Pull request merged**